File: NewsSystem.Api/Socket/SocketRequest.cs
namespace NewsSystem.Api.Socket
{
    using System.Collections.Generic;
    using System.Xml.Serialization;

    public class SocketRequest
    {
        public string EntityType { get; set; }

        public string Method { get; set; }

        public IDictionary<string, object> Params { get; set; }
    }
}

File: NewsSystem.Socket/src/newssystem/socket/commnication/SocketRequest.java
package newssystem.socket.commnication;

import java.util.Map;

public class SocketRequest {
    public String entityType;
    
    public String method;
    
    public Map<String, Object> params;
    
	public String getEntityType() {
		return entityType;
	}

	public void setEntityType(String entityType) {
		this.entityType = entityType;
	}

	public String getMethod() {
		return method;
	}

	public void setMethod(String method) {
		this.method = method;
	}

	public Map<String, Object> getParams() {
		return params;
	}

	public void setParams(Map<String, Object> params) {
		this.params = params;
	}
}
